consensus/state.go

```diff
@@ -110,7 +110,9 @@ func (consensus *Consensus) getAllNeighborsConsensusState() (*sync.Map, error) {
 // consensus height, or zero if no majority can be found
 func (consensus *Consensus) getNeighborsMajorityConsensusHeight() uint32 {
 	for i := 0; i < getConsensusStateRetries; i++ {
-		time.Sleep(getConsensusStateRetryDelay)
+		if i > 0 {
+			time.Sleep(getConsensusStateRetryDelay)
```

Comment:
How about move the sleep to end of `for` statement? Achieved the aim which ignore sleep of first neighbor but with no need for `if i>0` condition.

Reply:
This will make `continue` to continue immediately without retry delay

Reply:
How about defer time.Sleep?

Reply:
Defer only works when function returns, this is a for loop